### PR TITLE
remove ` /// <include file='` an `///<internalonly/>` comments

### LINKED ISSUE #167: remove ` /// <include file='` an `///<internalonly/>` comments
Comments like the following are no longer used:

 `  /// <internalonly />`

`/// <include file='doc\SystemInformation.uex' path='docs/doc[@for="SystemInformation.TerminalServerSession"]/*' />`

### PR DESCRIPTION
**Pull request by reaction1989**
fixes #167 

following RegExes were used:

^.*///<internalonly/>.*$\n
^.*/// <include file='.*$\n

Had to do datagridview manually because of the comments

### CODE REVIEW

**Comment by zsd4yr** [comment]
Looks like PropertyGrid.cs got messed up

File: src/System.Windows.Forms.Design.Editors/src/Misc/CommonUnsafeNativeMethods.cs [hunk 1]
```diff
@@ -18,22 +18,17 @@ internal class CommonUnsafeNativeMethods
         internal const int LOAD_LIBRARY_SEARCH_SYSTEM32 = 0x00000800;
 
         [DllImport(ExternDll.Kernel32, SetLastError = true, ExactSpelling = true, CharSet = CharSet.Ansi)]
-        [ResourceExposure(ResourceScope.Process)]
```
Comment (RussKie) [inline src/System.Windows.Forms.Design.Editors/src/Misc/CommonUnsafeNativeMethods.cs]: Why has this changed? This change relates to #321.
Please rebase on top of the latest master.

File: src/System.Windows.Forms/src/System/Windows/Forms/AxHost.cs [hunk 2]
```diff
@@ -4130,7 +4002,7 @@ private class OleInterfaces
             }
 
             // IPropertyNotifySink methods
-
+
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms/src/System/Windows/Forms/AxHost.cs]: please remove white spaces
```suggestion
```

File: src/System.Windows.Forms/src/System/Windows/Forms/Control.cs [hunk 3]
```diff
@@ -5124,8 +4931,8 @@ protected internal bool RenderRightToLeft
         ///     Helper method for retrieving an ActiveX property.  We abstract these
         ///     to another method so we do not force JIT the ActiveX codebase.
         /// </devdoc>
-        [ResourceExposure(ResourceScope.Process)]
-        [ResourceConsumption(ResourceScope.Process)]
+
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms/src/System/Windows/Forms/Control.cs]: Please remove the extra spaces

File: src/System.Windows.Forms/src/System/Windows/Forms/Control.cs [hunk 4]
```diff
@@ -5558,16 +5356,15 @@ protected internal bool RenderRightToLeft
...
         ///     Creates a Graphics for this control. The control's brush, font, foreground
         ///     color and background color become the default values for the Graphics.
         ///     The returned Graphics must be disposed through a call to its dispose()
         ///     method when it is no longer needed.  The Graphics Object is only valid for
         ///     the duration of the current window's message.
         /// </devdoc>
-        [ResourceExposure(ResourceScope.Process)]
-        [ResourceConsumption(ResourceScope.Process)]
+        
+
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms/src/System/Windows/Forms/Control.cs]:
```suggestion

```

File: src/System.Windows.Forms/src/System/Windows/Forms/Control.cs [hunk 5]
```diff
@@ -5576,13 +5373,12 @@ protected internal bool RenderRightToLeft
         }
        
 
-        [ResourceExposure(ResourceScope.Process)]
-        [ResourceConsumption(ResourceScope.Process)]
+        
+        
         internal System.Drawing.Graphics CreateGraphicsInternal() {
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms/src/System/Windows/Forms/Control.cs]: Please remove the extra lines

File: src/System.Windows.Forms/src/System/Windows/Forms/DataGridViewRowHeaderCell.cs [hunk 6]
```diff
@@ -185,8 +181,8 @@ private static Bitmap GetArrowStarBitmap(bool rightToLeft)
             return rightToLeft ? DataGridViewRowHeaderCell.LeftArrowStarBitmap : DataGridViewRowHeaderCell.RightArrowStarBitmap;
         }
 
-        [ResourceExposure(ResourceScope.Machine)]
-        [ResourceConsumption(ResourceScope.Machine)]
+
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms/src/System/Windows/Forms/DataGridViewRowHeaderCell.cs]: Please remove the extra lines

File: src/System.Windows.Forms/src/System/Windows/Forms/Design/ComponentEditorPage.cs [hunk 7]
```diff
@@ -153,13 +142,12 @@ public override bool AutoSize
...
         }
 
-        /// <include file='doc\ComponentEditorPage.uex' path='docs/doc[@for="ComponentEditorPage.Icon"]/*' />
         /// <devdoc>
         ///    <para>Gets or sets the icon for this page.</para>
         /// </devdoc>
         public Icon Icon {
-            [ResourceExposure(ResourceScope.Machine)]
-            [ResourceConsumption(ResourceScope.Machine)]
+
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms/src/System/Windows/Forms/Design/ComponentEditorPage.cs]: please remove the extra lines

File: src/System.Windows.Forms/src/System/Windows/Forms/Design/PropertyTab.cs [hunk 8]
```diff
@@ -22,19 +21,17 @@ public abstract class PropertyTab : IExtenderProvider {
...
 
         // don't override this. Just put a 16x16 bitmap in a file with the same name as your class in your resources.
-        /// <include file='doc\PropertyTab.uex' path='docs/doc[@for="PropertyTab.Bitmap"]/*' />
         /// <devdoc>
         ///    <para>Gets or sets a bitmap to display in the property tab.</para>
         /// </devdoc>
         public virtual Bitmap Bitmap {
-            [ResourceExposure(ResourceScope.Machine)]
-            [ResourceConsumption(ResourceScope.Machine)]
+
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms/src/System/Windows/Forms/Design/PropertyTab.cs]: please remove the extra lines

File: src/System.Windows.Forms/src/System/Windows/Forms/ErrorProvider.cs [hunk 9]
```diff
@@ -590,14 +571,13 @@ public class ErrorProvider : Component, IExtenderProvider, ISupportInitialize {
...
 
-        /// <include file='doc\ErrorProvider.uex' path='docs/doc[@for="ErrorProvider.Region"]/*' />
         /// <devdoc>
         ///     Create the icon region on demand.
         /// </devdoc>
         /// <internalonly/>
         internal IconRegion Region {
-            [ResourceExposure(ResourceScope.Machine)]
-            [ResourceConsumption(ResourceScope.Machine)]
+
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms/src/System/Windows/Forms/ErrorProvider.cs]: please remove extra lines

**Comment by zsd4yr** [comment]
@reaction1989 I am closing this PR due to inactivity. If you would like to resolve these merge conflicts and make this contribution, we are happy to take it 😄

**Pull request closed without merging**